### ISSUE TITLE
Create Social Media Accounts

### ISSUE THREAD

**Issue by ghost** [Utah-Node-Ninjas/unn-committee#8]
I added Utah Node Ninjas on my linkedin profile and was sad that Utah Node Ninjas wasn't in the dropdown.  I also just read that social media helps with getting sponsers.
- [x] twitter
- [ ] linkedin
- [x] facebook
- [x] youtube
- [x] google+

**Comment by ghost** [comment]
All of these are available on twitter, I vote for the second
utnode
utahnode +1
utahnodeninja

**Comment by derekargueta** [comment]
Currently all we have is a Facebook page https://www.facebook.com/utahnode.jsninjas

**Comment by derekargueta** [comment]
We also already have a gmail account set up at utahnodeninjas@gmail.com so Google+ and YouTube are _technically_ kinda there, just needs to be enabled

**Comment by ghost** [comment]
Thanks, Derek.  I would like to setup utahnodeninjas on twitter and linkedin and then close this issue.  We can open a separate issue for coordinating usage.  All in favor say +1.

**Comment by joeycozza** [comment]
+1

**Comment by ghost** [comment]
Twitter is done:
https://twitter.com/utahnodeninjas

LinkedIn requires an email with a business domain

**Comment by ghost** [comment]
waiting on #7 for email with our domain name, needed for linked in

**Comment by ghost** [comment]
I created the following on LinkedIn:

[Utah Node.js Ninjas Organizers (company)](https://www.linkedin.com/company/utah-node-js-ninjas)

[Utah Node.js Ninjas (group)](https://www.linkedin.com/groups/Utah-Nodejs-Ninjas-6943829)
